cypress.json

```diff
@@ -1,5 +1,6 @@
 {
   "video": false,
   "viewportWidth": 1200,
-  "viewportHeight": 600
+  "viewportHeight": 600,
+  "chromeWebSecurity": false
```

Comment:
Otherwise Cypress complains about CORS when we make calls to our backend on a different port from the UI:

```
Access to fetch at
'http://localhost:8080/v1/source/local-branches//Users/rudolfs/work/radicle-upstream/fixtures/empty-repo'

from origin
'http://localhost:50061'

has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present
on the requested resource. If an opaque response serves your needs, set the request's
mode to 'no-cors' to fetch the resource with CORS disabled.
```

This happens only when running tests in debug mode via `yarn test:debug`.

Reply:
Sweet I noticed this too!

Reply:
Thanks for fixing!